.readthedocs.yaml

```diff
@@ -0,0 +1,35 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
```

Comment:
However, this comes from the default template, the other `*.yaml` use `ubuntu-latest`. It may be worth considering using that to be a bit more uniform. It's okay like this as well, so I'll approve this PR.

Reply:
It might be better to use a fixed version so we don't have PR randomly failing, because there was an unrelated change somewhere